searx/engines/google_images.py

```diff
@@ -51,7 +51,7 @@ def request(query, params):
         search_options['tbs'] = time_range_custom_attr.format(start=start, end=end)
 
     if safesearch and params['safesearch']:
-        search_options['safe'] = 'active'
+        search_options['safe'] = 'on'
```

Comment:
"active" is used when using a browser, but "on" seems to work in this case.

Reply:
Fine, I have not looked at that part of the code, just having merged current master into my branch...